agent/command/git.go

```diff
@@ -199,7 +208,22 @@ func (opts cloneOpts) buildHTTPCloneCommand(forApp bool) ([]string, error) {
 	} else {
 		gitURL = thirdparty.FormGitURL(urlLocation.Host, opts.owner, opts.repo, opts.token)
 	}
-	clone := fmt.Sprintf("git clone %s '%s'", gitURL, opts.dir)
+
+	gitCommand := "git clone"
+	if opts.useScalar {
+		scalarAvailable, err := agentutil.IsGitVersionMinimumForScalar(thirdparty.RequiredScalarGitVersion)
+		if err != nil {
+			return nil, errors.Wrap(err, "checking git version")
```

Comment:
if we have an issue checking git version, should we just log and fall back to not available?

Reply:
done